Examples/test-suite/fortran/Makefile.in

```diff
@@ -0,0 +1,68 @@
+#######################################################################
+# Makefile for Fortran test-suite
+#######################################################################
+
+LANGUAGE	= fortran
+FC      	= @FC@
+SCRIPTSUFFIX= _runme.f90
+
+FORTRANOBJEXT = o
+
+FORTRAN_SO = @FORTRAN_SO@
+FCFLAGS = @FCFLAGS@
+CCSHARED=	@CCSHARED@
+
+srcdir         = @srcdir@
+top_srcdir     = @top_srcdir@
+top_builddir   = @top_builddir@
+abs_top_srcdir = @abs_top_srcdir@
+
+CPP_TEST_CASES = \
+	fortran_bindc
+
+CPP11_TEST_CASES = \
+	fortran_shared_ptr
+
+C_TEST_CASES = \
+	fortran_array_typemap \
+	fortran_bindc_c \
+	fortran_bindc_all \
+	fortran_enums \
+
+include $(srcdir)/failing-cpp-tests.mk
+include $(srcdir)/failing-c-tests.mk
+include $(srcdir)/../common.mk
+
+.SUFFIXES: .cpptest .ctest .multicpptest
+
+# Rules for the different types of tests
+%.cpptest:
+	$(setup)
+	+$(swig_and_compile_cpp)
+	$(run_testcase)
+
+%.ctest:
+	$(setup)
+	+$(swig_and_compile_c)
+	$(run_testcase)
+
+%.multicpptest:
+	$(setup)
+	@echo "*** Fortran multi tests are disabled; skipping '$*'"
+
+# Runs the testcase.
+run_testcase = \
+	if [ -f $(SCRIPTDIR)/$(SCRIPTPREFIX)$*$(SCRIPTSUFFIX) ]; then \
+	$(COMPILETOOL) $(FC) $(FCFLAGS) $(CCSHARED) $(SCRIPTDIR)/$(SCRIPTPREFIX)$*$(SCRIPTSUFFIX) -c; \
+	$(COMPILETOOL) $(FC) $(FCFLAGS) $(CCSHARED) -o $*_runme.exe $(SCRIPTPREFIX)$*_runme.@OBJEXT@ $(LIBPREFIX)$*$(FORTRAN_SO); \
+	env LD_LIBRARY_PATH=.:$$LD_LIBRARY_PATH $(RUNTOOL) ./$*_runme.exe; \
+	fi
+
```

Comment:
`make clean-fortran-test-suite` incorrectly cleans the *runme.f90 files.

Reply:
Oh the reason I didn't see this is because I never used an in-source build. :) Fixing.